### PR TITLE
Structuring intro patch

### PR DESCRIPTION
**Pull request by milstan**

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ const config = {
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
       colorMode: {
-        defaultMode: 'dark',
+        defaultMode: 'light',
```
Comment (m4rk055) [inline docusaurus.config.js:96]: Why changing to the light mode? Dark is more comfortable for long reads

File: src/pages/index.tsx [hunk 2]
```diff
@@ -43,7 +43,7 @@ function HeaderHero() {
             <GitHubButton
               // @ts-ignore
               className={styles.indexCtasGitHubButton}
-              href="https://github.com/blindnet-io/blindnet.dev"
+              href="https://github.com/blindnet-io"
```
Comment (m4rk055) [inline src/pages/index.tsx:46]:
```suggestion
              href="https://github.com/blindnet-io/blindnet.dev"
```

File: docs/02-tutorials/01-intro/index.mdx [hunk 3]
```diff
@@ -9,20 +9,16 @@ import { useCurrentSidebarCategory } from '@docusaurus/theme-common';
 
 # DevKit - Introductory Tutorial
 
-:::info
+The purpose of the **intro** tutorial is get you started quickly with blindnet DevKit.
 
-This tutorial doesn’t assume any existing knowledge of the blindnet devkit or even any privacy-related technologies.
+In few minutes you will be able to:
```
Comment (jboileau99) [inline docs/02-tutorials/01-intro/index.mdx:14]:
```suggestion
In a few minutes you will be able to:
```

**Review by m4rk055 — CHANGES_REQUESTED** [review]

File: src/css/custom.css [hunk 4]
```diff
@@ -6,31 +6,31 @@
...
 /* You can override the default Infima variables here. */
 :root {
-  --ifm-color-primary: #0c6a02;
-  --ifm-color-primary-dark: #0b5f02;
-  --ifm-color-primary-darker: #0a5a02;
-  --ifm-color-primary-darkest: #084a01;
-  --ifm-color-primary-light: #0d7502;
-  --ifm-color-primary-lighter: #0e7d02;
-  --ifm-color-primary-lightest: #108a03;
+  --ifm-color-primary: #a000c8;
```
Comment (m4rk055) [inline src/css/custom.css:9]: I find the green color much more comfortable. Should we take a vote?

image

image

File: docs/02-tutorials/04-connect-storage/index.mdx [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+# DevKit - Connect storage
+
+To take full advantage of the automation functionalities offered by blindnet devkit, you might consider connecting it with your storage.
+That way, you can automatically fetch the data follwoing a user's `ACCESS` request, or even automatically delete data should a `DELETE` request get accepted.
+Configuring a connector might save you a lot of time when there is a request to be processed.
+
+> :construction_worker: writing in progress
+
+A [Data Access Component](/docs/storage/data-access) is here to transform requests from the [Privacy Computation Engine](/docs/computation) to efficiently execute automated operations on your data storage.
```
[the remark this answers is not in the document]
Comment (milstan) [reply docs/02-tutorials/04-connect-storage/index.mdx:18]: This is an old text that I just displaced from where it was. I expect the storage part to change when @TheKinrar makes his commits.

File: docusaurus.config.js (hunk 1 above)
Comment (milstan) [reply docusaurus.config.js:96]: Really? Ok, we can leave it dark then.

Comment (milstan) [inline docusaurus.config.js:96]:
```suggestion
        defaultMode: 'dark',
```

File: src/css/custom.css (hunk 4 above)
Comment (milstan) [reply src/css/custom.css:9]: Im OK with a vote. @jboileau99 @TheKinrar what do you prefer?

Feel free to directly commit whatever emerges as preference.

Comment (m4rk055) [reply src/css/custom.css:9]: Third option is to use the [default theme](https://docusaurus.io/docs/configuration), which I prefer the most.

Comment (jboileau99) [reply src/css/custom.css:9]: I also prefer the default theme over the other two

Comment (milstan) [reply src/css/custom.css:9]: I updated the css with the default colors

**Review by m4rk055 — APPROVED** [review]

**Review by jboileau99 — APPROVED** [review]

**Pull request merged**